pkg/crd/markers/validation.go

```diff
@@ -72,6 +72,9 @@ var FieldOnlyMarkers = []*definitionWithHelp{
 
 	must(markers.MakeDefinition("nullable", markers.DescribesField, Nullable{})).
 		WithHelp(Nullable{}.Help()),
+
+	must(markers.MakeAnyTypeDefinition("kubebuilder:validation:Default", markers.DescribesField, Default{})).
```

Comment:
@marun before we release a release with this, do we want `:validation` in the middle, since this isn't actually validation (even though it's in the same place as validation)?  Not sure if that's confusing

Reply:
I'll remove.

Reply:
PR: https://github.com/kubernetes-sigs/controller-tools/pull/337